.gitignore

```diff
@@ -16,3 +16,6 @@ sassdoc-data.json
 backpack.iml
 yarn.lock
 /packages/*/changelog.md
+
+# Incase the script does not fully remove this we don't want to push to git
```

Comment:
Good to have this comment here, but I could foresee a situation in the future where somebody new doesn't know what we mean by 'the script', so I've expanded the comment to add some more context.

```suggestion
# On postpublish, we clone and run some scripts on the backpack-docs repo. This ensures we don't accidentally commit it.
```